### PR TITLE
Add the March 10, 2022 changelog

### LINKED ISSUE #1747: Release March 10, 2022 changelog

### LINKED ISSUE #1772: Changelog announcing GitHub Enterprise support
Part of https://github.com/gitpod-io/gitpod/issues/8501

### PR DESCRIPTION
**Pull request by mikenikles**
Close #1747.
Resolve #1772

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1749"></a>

### CODE REVIEW

File: src/lib/contents/changelog/2022-03-10.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: TODO
```
Comment (Cahllagerfeld) [inline src/lib/contents/changelog/2022-03-10.md:2]: `title` & `excerpt` are still on TODO, just as a side-note before merging :D

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]

**Comment by mikenikles** [comment]
Waiting for @jankeromnes to provide a header image ([internal conversation](https://gitpod.slack.com/archives/C022HAT0H5Z/p1646904979770979?thread_ts=1646875681.535049&cid=C022HAT0H5Z)).

Once available, this PR is ready to be merged.

**Comment by Cahllagerfeld** [comment]
@jankeromnes Could you please add the header image, so we can publish this delayed Changelog 😬

**Review by Cahllagerfeld — APPROVED** [review]
Thanks for adding the image @jldec

**Pull request merged**